### PR TITLE
lumo 1.10.1: fix missing dependency

### PR DESCRIPTION
**Pull request by hlolli**
###### Motivation for this change

Lumo is currently failing on linux https://hydra.nixos.org/build/112812334 and a fix was merged today

this merge includes darwin support https://hydra.nixos.org/build/113308416 which is failing, I hope darwin can safely be left out here without affecting the linux support?

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by worldofpeace** [comment]
Can you please cherry-pick the actual commit https://github.com/NixOS/nixpkgs/commit/ad566721969e9e765cabfaa6ef972713260757c0, and not the merge commit?

**Comment by hlolli** [comment]
@worldofpeace I've been in the habit of squashing commits on the gihub.com ui, so I messed up a bit by not squashing, so I thought the merge would make sense in this case as it would encompass the changes from 2 commits. Now I've cherry picked the 2 commits, the other one titled terribly "cleanup". It's a small mess, I hope I'm forgiven :)

**Comment by worldofpeace** [comment]
@hlolli You can do an interactive rebase and squash that cleanup commit into the first, and make sure the first commit has the  

```
(cherry picked from commit ef712ff)
(cherry picked from commit ad56672)
```
 in the body.

**Comment by hlolli** [comment]
@worldofpeace done

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build lemo

File: pkgs/development/interpreters/clojurescript/lumo/package.json [hunk 1]
```diff
@@ -1,40 +1,49 @@
...
+    "colors": "^1.3.3",
+    "cross-env": "7.0.0",
     "death": "^1.1.0",
-    "flow-bin": "0.85.0",
+    "flow-bin": "0.118.0",
     "google-closure-compiler-js": "20170910.0.1",
+    "glob": "^7.1.3",
+    "gunzip-maybe": "^1.4.1",
+    "insert-module-globals": "^7.2.0",
     "jszip": "github:anmonteiro/jszip#patch-1",
```
Comment (Mic92) [inline pkgs/development/interpreters/clojurescript/lumo/package.json:30]: This repository is no longer reachable.

**Comment by hlolli** [comment]
@Mic92 sorry about being a bit clueless, I didn't remember my changes being merged already :) so I changed the commit message to just refer to a fix. I tested locally and the binary compiles and runs fine, need to research if this patch was of any importance, but afaik, it wasn't.

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**